src/main/webapp/app/iris/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.ts

```diff
@@ -72,6 +74,23 @@ export class ExerciseChatWidgetComponent implements OnInit, OnDestroy, AfterView
         }, 500);
     }
 
+    loadFirstMessage(): void {
+        if (this.messages.length === 0) {
+            const firstMessageContent = {
+                textContent: "Hi, I'm Iris! I can help you with your programming exercise. You can know more about me here.",
+                type: IrisMessageContentType.TEXT,
+            } as IrisMessageContent;
+
+            const firstMessage = {
+                sender: IrisSender.LLM,
+                id: 1,
```

Comment:
does it work if we use `0` instead of `1`? In case we have a conflict with the server's response which may have id 1 also

Reply:
For the red dot maybe not, and I'll try `id: 0`